docs/docs_toolkit.md

```diff
@@ -1 +1,179 @@
-# Documentation tookit
\ No newline at end of file
+# Documenting your project
+
+This section helps you add documentation to your Ansible project or improve your existing documentation.
+The onboarding [project-template](https://github.com/ansible-community/project-template)
+in GitHub gives you the scaffolding files to start your documentation effort. 
+
+## Creating new documentation projects
+
+Ansible projects starting a new documentation effort should follow these guidelines:
+
+* Markdown is simple, ubiquitous, and familiar to most contributors. For these reasons
+ the Ansible community team recommends markdown as the format for documentation.
+* [mkdocs](https://www.mkdocs.org/getting-started/) is the recommended tool for building 
+the documentation from markdown.
+* [ReadtheDocs](https://readthedocs.org/) is the recommended place to publish documentation
+ for Ansible projects.
+* The [Ansible style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html) 
+is available to help ensure your project is well-written and follows technical documentation guidelines.
+
+Follow these steps to set up your documentation if you do not use the GitHub project-template:
+
+1. Create a ``/docs`` directory in your project repository. 
+2. Follow the proposed [documentation structure](https://github.com/ansible-community/project-template/tree/main/docs)
+ from the project-template to ensure you include all the important guides common
+to all projects.
+3. Add a [`mkdocs.yml`](https://github.com/ansible-community/project-template/blob/main/mkdocs.yml) to
+ your project root directory and modify the `nav` section to match your proposed navigation/set of guides.
+4. Create [``.readthedocs.yml``](https://github.com/ansible-community/project-template/blob/main/.readthedocs.yaml) in your project root directory.
+5. Coordinate with the [Ansible community team](https://docs.ansible.com/ansible/latest/community/communication.html#forum) to set up automated publishing and to have your documentation added to the Ansible ecosystem page.
```

Comment:
```suggestion
5. Ask the [Ansible community team](https://docs.ansible.com/ansible/latest/community/communication.html#forum) to help set up server-side configuration on Read The Docs and add your project to the Ansible namespace.
```

Reply:
This last bit seems a little risky to me. Does this onboarding apply only to projects in the Ansible org?